apps/storybook/components/funke/BiometricsModal.stories.tsx

```diff
@@ -0,0 +1,87 @@
+        fill={theme['primary-500'].val}
+      />
+    </Svg>
+  )
+}
+
+const BiometricsModal = ({ onAcceptBiometrics, open, onOpenChange }: BiometricsModalProps) => {
+  return (
+    <Sheet
+      open={open}
+      onOpenChange={onOpenChange}
+      // FIXME: hardcoded for good fit in storybook, but this probably won't work on mobile...
+      snapPoints={[520]}
+      snapPointsMode="constant"
+      disableDrag
+      dismissOnOverlayPress={false}
+    >
+      <Sheet.Overlay
+        animation="lazy"
+        enterStyle={{ opacity: 0 }}
+        exitStyle={{ opacity: 0 }}
+        backgroundColor="$black"
+        opacity={0.2}
+      />
+      <Sheet.Frame borderRadius="$8" maxHeight={360} width="90%" alignSelf="center">
+        <YStack flex-1 justifyContent="space-between" overflow="hidden">
+          <Stack backgroundColor="$primary-100" height={175} justifyContent="center" alignItems="center">
+            <FingerprintIcon />
+          </Stack>
+          <YStack p="$2" px="$4" gap="$3" height={100}>
+            <Heading variant="h2">Biometrics is required</Heading>
+            <Paragraph variant="text">
+              Securing with your biometrics is required for receiving your digital identity
+            </Paragraph>
+          </YStack>
+          <Button.Solid onPress={onAcceptBiometrics} mx="$4" mb="$4">
+            Activate Biometrics
+          </Button.Solid>
+        </YStack>
+      </Sheet.Frame>
```

Comment:
So ideally we have the sheet itself take the height of the content. And then we use padding around the Icon to scale the light-shaded section. 

Then use 'gap' to separate the sections from eachother. 

The text section can then also take the height based on the content. Using fixed heights will mess it up if your text is longer on smaller devices (or because of text changes later.

Reply:
Yeah I need to look into this again on a proper RN screen. Sheet and Storybook was giving a lot of troubles.